### ISSUE TITLE
VeraCrypt 1.23 and 1.24 not responding to some keys

### ISSUE THREAD

**Issue by gitthepie** [veracrypt/VeraCrypt#660]
I've noticed that on one of my computers, VeraCrypt (Linux version 1.23 and 1.24) doesn't respond to the enter or tab keys. I now have to mount and unmount containers by clicking ok with my mouse. I have another computer with the same version of Linux but it doesn't have the problem. The problem probably came up about a month ago, and since I haven't updated lately (except to 1.24 to try to fix the problem), it probably wasn't caused by a new version of VeraCrypt. Does anyone know what might be causing my problem?

**Comment by idrassi** [comment]
Usually such UI issues are linked to the Window Manager used under Linux and it compatibility with the GTK libraries that VeraCrypt is linked against.
Did you install VeraCrypt using the same package on both Linux machines? Maybe on one machine you used the generic installer while on the other you used the native installation package.Using the native installation package is better since it will install a binary that is linked with native wxWidgets library of the Linux distribution you are using, and this offer better compatibility chances.

**Comment by gitthepie** [comment]
Which one is the generic installer? It looks like I used veracrypt-1.23-setup.tar.bz2 for version 1.23, but I noticed that there is a .deb for version 1.24. However, I still had the problem on the buggy machine after removing VeraCrypt 1.23 and installing VeraCrypt 1.24 using the above files. I did notice, however, that the settings for 1.24 were identical to those of 1.23, so perhaps 1.23 wasn't uninstalled cleanly?

Do you think cleanly removing VeraCrypt and reinstalling using the correct installer (whichever one that might be) might help? I use VeraCrypt regularly, so this problem is a bit more than a minor annoyance.

**Comment by gitthepie** [comment]
All right, so I uninstalled VeraCrypt using 2 different methods:

1. sudo veracrypt-uninstall.sh
2. sudo apt-get purge --autoremove veracrypt

I did #2 because I found a package named veracrypt on the system.

After the above, I rebooted and reinstalled VeraCrypt using veracrypt-1.24-Update4-Ubuntu-18.04-amd64.deb. This should be the native installation package you mentioned. But I still have the problem.

What should my next step be?
